Classes/Middleware/EventDefinitionRegistererMiddleware.php

```diff
@@ -0,0 +1,40 @@
+<?php
+declare(strict_types=1);
+
+/*
+ * Copyright (C) 2018
```

Comment:
The year is not correct :p

```suggestion
 * Copyright (C) 2020
```